demisto_sdk/commands/format/update_layout.py

```diff
@@ -211,3 +218,48 @@ def layoutscontainer__arguments_to_remove(self):
                 second_level_args[kind] = set(self.data[kind].keys()) - set(kind_schema)
 
         return first_level_args, second_level_args
+
+    @staticmethod
+    def remove_copy_and_dev_suffixes_from_str(field_name):
+        return field_name.replace('_dev', '').replace('_copy', '')
```

Comment:
please consider python's [built in](https://peps.python.org/pep-0616/) `removesuffix`, to avoid odd cases where _dev is in the name but not in the suffix.